src/hooks/useRoom.ts

```diff
@@ -22,6 +22,7 @@ export const useRoom = ({ room, venueName }: UseRoomProps) => {
 
   const roomUrl = room.url;
 
+  // @debt This selector relies on all venues in firebase being loaded into memory.. not very efficient
   const venues = useSelector(orderedVenuesSelector);
```

Comment:
[note, ?OOS?] `useRoom` is used by:

- `src/components/molecules/EventDisplay/EventDisplay.tsx`
- `src/components/templates/PartyMap/components/RoomAttendance.tsx`
- `src/components/templates/PartyMap/components/Map/MapRoom.tsx`
- `src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx`

Reply:
To bring some context. It's used to decide if the provided string is a venue or an external link

Reply:
OOS, will handle this in a followup PR.

Reply:
> To bring some context. It's used to decide if the provided string is a venue or an external link

Surely we can figure that out in a simpler way.